build_wasm.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies] doesn't work for avoiding compilation dev-dependecies for wasm32
+cd ./token-lending/program 
+
+# deisable dependecies
+sed -i '' "s/proptest/#proptest/" Cargo.toml
+sed -i '' "s/solana-program-test/#solana-program-test/" Cargo.toml
+
+cargo build --lib --release --target wasm32-unknown-unknown
```

Comment:
I would suggest to use [Platform specific dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies) instead of bash script.

Reply:
You're right. I've missed it.

Reply:
The issue can be fixed by adding `resolver = "2"` line to a workspace `Cargo.toml`

- https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions
- https://github.com/rust-lang/cargo/issues/10749#issuecomment-1152989102

Reply:
fixed